Description: Refer to encryption/decryption scripts from Filter::decrypt(3pm)
Author: Colin Watson <cjwatson@debian.org>
Bug-Debian: http://bugs.debian.org/566739
Forwarded: no
Last-Update: 2010-05-26

Index: b/decrypt/decrypt.pm
===================================================================
--- a/decrypt/decrypt.pm
+++ b/decrypt/decrypt.pm
@@ -104,6 +104,11 @@
 you could try using the unexec/undump method. See the Perl FAQ for
 further details.
 
+=head1 EXAMPLES
+
+See /usr/share/doc/libfilter-perl/examples/decrypt/ for examples of
+encryption and decryption scripts using this filter.
+
 =head1 AUTHOR
 
 Paul Marquess 
